src/plugins/intel_cpu/src/transformations/snippets/tpp/pass/scalar_to_scalar_tpp.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/itt.hpp"
+#include "scalar_to_scalar_tpp.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "snippets/op/scalar.hpp"
+#include "transformations/snippets/tpp/op/scalar.hpp"
+#include "transformations/snippets/tpp/op/modifiers.hpp"
+#include "snippets/lowered/port_connector.hpp"
+
+
+namespace ov {
+namespace intel_cpu {
+namespace tpp {
+namespace pass {
+
+ScalarToScalarTPP::ScalarToScalarTPP() {
+    MATCHER_SCOPE(ScalarToScalarTPP);
+
+    auto snippets_scalar = ov::pass::pattern::wrap_type<ov::snippets::op::Scalar>();
+
+
+    auto callback = [=](ov::pass::pattern::Matcher& m) {
+        OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "ov::intel_cpu::pass::ScalarToScalarTPP")
+        const auto node = ov::as_type_ptr<ov::snippets::op::Scalar>(m.get_match_root());
+        OPENVINO_ASSERT(node, "Failed to obtain a valid Scalar Op in ScalarToScalarTPP");
+        size_t num_connected_tpp = 0;
+        const auto& target_ins = node->get_output_target_inputs(0);
+        for (const auto& in : target_ins) {
+            if (dynamic_cast<tpp::modifier::TensorProcessingPrimitive*>(in.get_node()))
+                num_connected_tpp++;
+        }
+        if (num_connected_tpp == 0)
+            return false;
+        OPENVINO_ASSERT(num_connected_tpp == target_ins.size(), "Either all or none Scalar outputs should be TPP");
+
+        const auto& tpp_scalar = std::make_shared<tpp::op::Scalar>(*node);
+        tpp_scalar->set_friendly_name(node->get_friendly_name());
+        ov::replace_node(node, tpp_scalar);
```

Comment:
Please copy rt_info too

Reply:
I'm not sure that it's safe because `PortDescriptors` are stored in `rt_info`...
If `Scalar` had `PortDescriptors`, `ScalarTPP` gets them.
We should be accurate there

Reply:
@a-sidorova, do you have a specific example in mind?
I would also expect that the rt_info should be copied. From the pipeline's point of view, TPP operation should be indistinguishable from other ops, so if snippets::Scalar has a certain rt_inifo, then tpp::Scalar should have it too.

Reply:
Recently I worked with GN and PortDescriptor in `rt_info` and recalled that `PD` is [not](https://github.com/openvinotoolkit/openvino/blob/149381ef8cda82b42d5dfaa0a4ab81700e0a7322/src/common/snippets/include/snippets/lowered/port_descriptor.hpp#L110) copyable in `rt_Info`))
So everything is fine: we should copy `rt_info` here and it's safe